src/test/java/com/zufar/onlinestore/cart/ShoppingSessionItemDtoConverterTest.java

```diff
@@ -0,0 +1,112 @@
+package com.zufar.onlinestore.cart;
+
+
+import com.zufar.onlinestore.cart.converter.ShoppingSessionItemDtoConverter;
+import com.zufar.onlinestore.cart.dto.ShoppingSessionItemDto;
+import com.zufar.onlinestore.cart.entity.ShoppingSession;
+import com.zufar.onlinestore.cart.entity.ShoppingSessionItem;
+import com.zufar.onlinestore.product.entity.ProductInfo;
+import org.junit.jupiter.api.Test;
+import org.mapstruct.factory.Mappers;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+@SpringBootTest
+public class ShoppingSessionItemDtoConverterTest {
+
+    @Autowired
+    private ShoppingSessionItemDtoConverter mapper
+            = Mappers.getMapper(ShoppingSessionItemDtoConverter.class);
+
+    @Test
+    public void mapping_ShoppingSessionItem_to_ShoppingSessionItemDto() {
+        ShoppingSessionItem shoppingSessionItem = createShoppingSessionItem();
+
+        ShoppingSessionItemDto dto = mapper.toDto(shoppingSessionItem);
+
+        assertThat(dto.id(), is(shoppingSessionItem.getId()));
+        assertThat(dto.productsQuantity(), is(shoppingSessionItem.getProductsQuantity()));
+        assertThat(dto.productInfo().id(), is(shoppingSessionItem.getProductInfo().getProductId()));
+        assertThat(dto.productInfo().quantity(), is(shoppingSessionItem.getProductInfo().getQuantity()));
+        assertThat(dto.productInfo().name(), is(shoppingSessionItem.getProductInfo().getName()));
+        assertThat(dto.productInfo().description(), is(shoppingSessionItem.getProductInfo().getDescription()));
+        assertThat(dto.productInfo().active(), is(shoppingSessionItem.getProductInfo().getActive()));
+        assertThat(dto.productInfo().priceDetails().price(), is(shoppingSessionItem.getProductInfo().getPrice()));
+        assertThat(dto.productInfo().priceDetails().currency(), is(shoppingSessionItem.getProductInfo().getCurrency()));
+    }
+
+    private ShoppingSessionItem createShoppingSessionItem() {
+        UUID id = UUID.fromString("f8c3de3d-1fea-4d7c-a8b0-29f63c4c3454");
+        ShoppingSession shoppingSession = createShoppingSession();
+        ProductInfo productInfo = createProductInfo();
+        Integer productsQuantity = 10;
+        Integer version = 155;
```

Comment:
this field is managed by Spring Date, I think it should not be used in tests, since we can't influence it in any way.